### DISCUSSION TITLE
Lost folder with docker-compose.yml

### DISCUSSION THREAD

**Discussion by medicus07** [teslamate-org/teslamate#3064] [Q&A]
Hi there,

I have the following problem: I wanted to update my Teslamate docker and realized that I have lost my folder with the docker-compose.yml in it. What to do... I can't backup and update anymore...

**Comment by cwanja** [comment]
Are you running on a Linux machine? If so, run the following commands

```
cd ~/
find . -name "docker-compose.yml"

```

**Reply by medicus07** [reply]
I´m using a Synology NAS (DS718+)... I´ve searched for the compose folder and the dml with no luck. I'm afraid the folder was deleted by an update.

**Comment by medicus07** [comment]
Anyone?

**Reply by markusdd** [reply]
the compose yml only contains your start parameters. was it heavily modified? otherwise just download a new one.
